doc/api/util.md

```diff
@@ -398,6 +398,83 @@ util.inspect.defaultOptions.maxArrayLength = null;
 console.log(arr); // logs the full array
 ```
 
+## util.promisify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function}
+
+Takes a function following the common Node.js callback style, i.e. taking a
+`(err, value) => ...` callback as the last argument, and returns a version
+that returns promises.
```

Comment:
Do you mean Sync as in calls the callback right away or as in never calls the callback? In the latter case, my expectation would be that the promise is neither resolved nor rejected - that seems well-defined.